### DISCUSSION TITLE
Multiple individual Status pages?

### DISCUSSION THREAD

**Discussion by edwardcox** [upptime/upptime#592] [General] [answered]
Hi All, hope this isn't a stupid question. Personally Upptime is awesome. 

However.. is it / would it be possible to have individual Status Pages for each 'site' being monitored? 
Would I need to setup multiple repos? Can you duplicate a repo and rename it?
Then using custom DNS CNAME for each of them I could achieve:
site1status1.mydomain.com
site2status2.mydomain.com

The main purpose being to keep each clients' Status Page separately and not all clustered together.

Any suggestions please?
Edward

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, the only way to currently do this is via multiple repositories, perhaps each with its own CNAME. It's a useful feature to have!

**Comment by edwardcox** [comment]
Thank you Annand! I appreciate your reply. Is there a way to ‘duplicate’ my
current working repository? Or do I just Add a new one as per instructions?

Thank you,

Edward


From: Anand Chowdhary ***@***.***> ***@***.***>
Reply: upptime/upptime
***@***.***>
***@***.***>
Date: 14 June 2022 at 3:55:56 pm
To: upptime/upptime ***@***.***>
***@***.***>
Cc: Edward Cox ***@***.***> ***@***.***>, Author
***@***.***> ***@***.***>
Subject:  Re: [upptime/upptime] Multiple individual Status pages?
(Discussion #592)

Yes, the only way to currently do this is via multiple repositories,
perhaps each with its own CNAME. It's a useful feature to have!

—
Reply to this email directly, view it on GitHub
<https://github.com/upptime/upptime/discussions/592#discussioncomment-2944076>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AACE3SZZJKUR4AMSV3WZZZTVPANGXANCNFSM5YTMTUJA>
.
You are receiving this because you authored the thread.Message ID:
***@***.***>

**Comment by AnandChowdhary** [comment]
You can use the GitHub import feature and that should duplicate the repository: https://github.com/new/import
